torchgen/api/translate.py

```diff
@@ -93,6 +95,7 @@ def translate(
     *,
     method: bool = False,
     allow_expensive_conversions: bool = False,
+    allow_tensor_const_cast: bool = False,
```

Comment:
In general we allow `Tensor&` to `const Tensor&` casts as inside the argument code mutable tensor references, so it's not clear to me that this is warranted; we should just always allow the const cast.

Reply:
good point!